packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -1056,3 +1057,14 @@ export function getClientNameOverride(
 ): string | undefined {
   return getScopedDecoratorData(context, clientNameKey, entity, languageScope);
 }
+
+const hasJsonConverterKey = createStateSymbol("hasJsonConverterKey");
+export const $hasJsonConverter: DecoratorFunction = (
+  context: DecoratorContext,
+  entity: Model,
+  scope?: LanguageScopes
+) => {
+  setScopedDecoratorData(context, $hasJsonConverter, hasJsonConverterKey, entity, true, scope);
```

Comment:
Do you even need to set the data then if you are never gonna read it

Reply:
you are right, it can just be empty